runelite-client/src/main/java/net/runelite/client/menus/MenuManager.java

```diff
@@ -269,11 +273,12 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 			}
 		}
 
-		String target = event.getMenuTarget();
+		// removes bounty hunter emblem tag and tier from player name, e.g. username<img=20>5 -> username
+		String targetWithoutBHEmblemAndTier = BOUNTY_EMBLEM_TAG_AND_TIER_REGEXP.matcher(event.getMenuTarget()).replaceAll("");
```

Comment:
I think leaving this named `target` is fine.
```suggestion
		String target = BOUNTY_EMBLEM_TAG_AND_TIER_REGEXP.matcher(event.getMenuTarget()).replaceAll("");
```